.github/workflows/ci.yaml

```diff
@@ -1,7 +1,9 @@
 name: integration and other tests
-
 on:
   workflow_call:
+  push:
```

Comment:
i am unsure that this is needed. we already run these tests as part of the release to edge workflow [here](https://github.com/canonical/mysql-router-k8s-operator/blob/main/.github/workflows/release.yaml#L23-L26)

Reply:
Fixed in the last commit.